package.json

```
@@ -1,42 +1,50 @@
 {
-  "name": "grenoble-hands-on-react",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
-    "bulma": "^0.9.1",
-    "history": "^5.0.0",
-    "leaflet": "^1.7.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-router": "^6.0.0-beta.0",
-    "react-router-dom": "6.0.0-beta.0",
-    "react-scripts": "4.0.0"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
+    "name": "grenoble-hands-on-react",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@testing-library/jest-dom": "^5.11.4",
+        "@testing-library/react": "^11.1.0",
+        "@testing-library/user-event": "^12.1.10",
+        "@types/jest": "^26.0.19",
+        "@types/node": "^14.14.16",
+        "@types/react-dom": "^17.0.0",
+        "bulma": "^0.9.1",
+        "history": "^5.0.0",
+        "leaflet": "^1.7.1",
+        "react": "^17.0.1",
+        "react-dom": "^17.0.1",
+        "react-router": "^6.0.0-beta.0",
+        "react-router-dom": "6.0.0-beta.0",
+        "react-scripts": "4.0.0-next.98",
```

La version `4.0.1` permet d’utiliser la version `4.1.x` de Typescript.
```suggestion
        "react-scripts": "4.0.1",
```